eva.py

```diff
@@ -137,15 +137,30 @@ def single2list(item, numpy_array=False):
     return output
 
 
+def aep_to_ari(aep):
+    """Convert from aep (%) to ari (years)
+    
+    Details: http://www.bom.gov.au/water/designRainfalls/ifd-arr87/glossary.shtml
+    """
+    
+    assert aep < 100, "aep to be expressed as a percentage (must be < 100)"
+    aep = aep/100
+    
+    return 1 / (-np.log(1-aep))
+
+
+def ari_to_aep(ari):
```

Comment:
The `ari_to_aep` function isn't actually used anywhere in the script. Were you thinking it would be nice to just have it in this script in case someone needs a function to do the reverse conversion?

Reply:
That's correct - it's not used anywhere but I included it in case we want to apply the reverse conversion somewhere in future. If you wanted to we could remove it as it's in this branch that can be referred to later?

Reply:
That's cool - I don't think it hurts to leave it in the script. I'll go ahead and merge :smile: